index.html

```diff
@@ -133,6 +150,7 @@ <h4 class="title--last">Custom Design designers</h4>
       </div>
 
       <section class="marketing-strategies">
+        <a name="customers"></a>
```

Comment:
Do not insert a separate a tag.
name attracted is outdated

Just add id to a section
`<section id="customers" class="marketing-strategies">`

Reply:
completed